ob-pattern-1/values.yaml

```diff
@@ -138,6 +138,10 @@ wso2:
       gateway:
         hostname: wso2-obam-gateway
 
+nfs:
+  # StorageClass name of the NFS
+  storageClassName: "nfs"
+
```

Comment:
Can we follow the same structure as in https://github.com/wso2/kubernetes-apim/blob/cdb7cd76de45f6b58d6259da162286a98078253d/advanced/am-pattern-1/values.yaml#L30

Reply:
+1 @Aaquiff. Yes, @kdanW I think we should follow the same structure.

Please consider this when you work on the next PR, in which you are to add support for external NFS server support.

Reply:
@chirangaalwis yes i will follow this structure when adding support for external NFS server